src/App.tsx

```diff
@@ -1,26 +1,261 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+import {
+  addTodo,
+  deleteTodo,
+  getTodos,
+  updateTodo,
+  USER_ID,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
+import { FilterType } from './types/filterType';
+import { ErrorMessages } from './types/errorMessages';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<ErrorMessages | string>(
+    ErrorMessages.none,
+  );
+  const [filterType, setFilterType] = useState<FilterType>(FilterType.all);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [deletedTodosId, setDeletedTodosId] = useState<number[]>([]);
+  const [toggledTodosId, setToggledTodosId] = useState<number[]>([]);
+  const errorTimer = useRef<number | null>(null);
+
+  const completedTodos = useMemo(
+    () => todos.filter(todo => todo.completed),
+    [todos],
+  );
+
+  const handleErrorMessage = (message: ErrorMessages) => {
+    setErrorMessage(message);
+
+    if (errorTimer.current) {
+      clearTimeout(errorTimer.current);
+    }
+
+    if (message !== ErrorMessages.none) {
+      errorTimer.current = window.setTimeout(() => {
+        setErrorMessage(ErrorMessages.none);
+        errorTimer.current = null;
+      }, 3000);
+    }
+  };
+
+  const getAllTodos = async () => {
+    try {
+      const allTodos = await getTodos();
+
+      setTodos(allTodos);
+    } catch {
+      handleErrorMessage(ErrorMessages.todosLoadError);
+    }
+  };
+
+  useEffect(() => {
+    getAllTodos();
+  }, [])
```

Comment:
Fix the linter errors 
<img width="924" height="208" alt="image" src="https://github.com/user-attachments/assets/cb90f43d-c3a1-414e-985d-afe4af7e10d9" />